doc/getting_started.rst

```diff
@@ -18,3 +21,15 @@ Currently, the best way to install RSMTool is by using the ``conda`` package man
 
 RSMTool can also be downloaded directly from
 `GitHub <http://github.com/EducationalTestingService/rsmtool>`_.
+
+Installing with pip
+-------------------
+
+You can also ``pip`` to install RSMTool instead of ``conda``. To do so, simply run::
```

Comment:
"use `pip`"?

Reply:
I am going to fix this real quick and merge this PR.